unlock-protocol-com/src/registerServiceWorker.ts

```diff
@@ -20,12 +20,15 @@ const isLocalhost = Boolean(
     )
```

Comment:
Are these changes relevant here?

Reply:
Not directly related to navbar. I'm moving js files to ts slowly as not to overwhelm with big PR and making sure nothing breaks.